### PR TITLE
br: add documents for s3-multi-part-size

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11183
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1815,3 +1815,13 @@ To reduce write latency and avoid frequent access to PD, TiKV periodically fetch
...
 + In the **TiKV-RAW** \> **Causal timestamp** panel in Grafana, **TSO batch size** is the number of locally cached timestamps that has been dynamically adjusted according to the application workload. You can refer to this metric to adjust `renew-batch-min-size`.
 + Default value: `100`
+
+### `s3-multi-part-size` <span class="version-mark">New in v5.3.2</span>
+
+> **Note:**
+>
+> This configuration is introduced to address backup failures caused by S3 rate limiting. This problem has been fixed by [refining the backup data storage structure](/br/backup-and-restore-design.md#backup-file-structure) in v6.1.1, v6.2.0, and later versions. Therefore, this configuration is now deprecated and is no longer recommended.
```
Comment (ran-huang) [inline tikv-configuration-file.md:1823]: I think simply saying that this config is deprecated is enough for v6.3 and dev users.

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1815,3 +1815,13 @@ To reduce write latency and avoid frequent access to PD, TiKV periodically fetch
...
+
++ The part size used when you perform multipart upload to S3 during backup. You can adjust this parameter to control the number of requests sent to S3.
```
Comment (ran-huang) [inline tikv-configuration-file.md:1825]:
```suggestion
+ The part size used when you perform multipart upload to S3 during backup. You can adjust the value of this configuration to control the number of requests sent to S3.
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1815,3 +1815,13 @@ To reduce write latency and avoid frequent access to PD, TiKV periodically fetch
...
++ If data is backed up to S3 and the backup file is larger than the value of this configuration item, [multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html) is automatically enabled. Based on the compression ratio, the backup file generated by a 96-MB Region is approximately 10 to 30 MB.
```
Comment (ran-huang) [inline tikv-configuration-file.md:1826]:
```suggestion
+ If data is backed up to S3 and the backup file is larger than the value of this configuration item, [multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html) is automatically enabled. Based on the compression ratio, the backup file generated by a 96-MB Region is approximately 10 MB to 30 MB.
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1815,3 +1815,13 @@ To reduce write latency and avoid frequent access to PD, TiKV periodically fetch
...
++ Default value: 5 MB
```
Comment (ran-huang) [inline tikv-configuration-file.md:1827]: The Chinese version does not have a space ("5MB").

File: tikv-configuration-file.md (hunk 1 above)
Comment (shichun-0415) [reply tikv-configuration-file.md:1823]: @IANTHEREAL I think it makes sense. What do you think?

Comment (IANTHEREAL) [reply tikv-configuration-file.md:1823]: just describe the fact

**Review by IANTHEREAL — APPROVED** [review]

File: tikv-configuration-file.md (hunk 4 above)
Comment (shichun-0415) [reply tikv-configuration-file.md:1827]: Fixed in [2d0a1a0](https://github.com/pingcap/docs-cn/pull/11183/commits/2d0a1a09187030312af1d2ce4dc4b2ba2ea7bb36)

File: tikv-configuration-file.md (hunk 1 above)
Comment (shichun-0415) [reply tikv-configuration-file.md]: Updated the doc in [ef08e0e](https://github.com/pingcap/docs/pull/10316/commits/ef08e0eed15d6d7d3e7c72978f7a724cafd41473). PTAL @ran-huang

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1815,3 +1815,13 @@ To reduce write latency and avoid frequent access to PD, TiKV periodically fetch
...
+
+### `s3-multi-part-size` <span class="version-mark">New in v5.3.2</span>
+
+> **Note:**
+>
+> This configuration is introduced to address backup failures caused by S3 rate limiting. This problem has been fixed by [refining the backup data storage structure](/br/backup-and-restore-design.md#backup-file-structure). Therefore, this configuration is deprecated from v6.1.1 and is no longer recommended.
+
++ The part size used when you perform multipart upload to S3 during backup. You can adjust the value of this configuration to control the number of requests sent to S3.
++ If data is backed up to S3 and the backup file is larger than the value of this configuration item, [multipart upload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html) is automatically enabled. Based on the compression ratio, the backup file generated by a 96-MB Region is approximately 10 MB to 30 MB.
++ Default value: 5 MB
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1827]:
```suggestion
+ Default value: 5MiB
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**